lib/encoder_impl.cc

```diff
@@ -443,7 +464,7 @@ void encoder_impl::prepare_group4a(void) {
 		((toffset>0?0:1)<<5)|((abs(toffset*2))&0x1f);
 }
 
-// for now single-group only
+// TMC Alert-C - http://tinyurl.com/alert-c-coding-handbook
```

Comment:
The full URL might be more stable. (also it looks like this is not pointing where it is supposed to be)